CONTRIBUTING.md

```diff
@@ -96,11 +95,9 @@ The subject contains succinct description of the change:
 * no dot (.) at the end
 
 ### Body
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+Wrap the body at 72 characters whenever possible (for example, don't modify backtraces to follow this rule). Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
```

Comment:
Are backtraces even a thing in commit messages?

I think that better example would be a long link